File: Basics/sumOfN.cpp
#include <iostream>
using namespace std;

int sumOfN(int n){
    int sum = 0;
    for(int i =0; i <=n; i++){
        sum += i;
    }
    return sum;
}

int main(){
    int n ;
    cout<< "enter the number n : "<<endl;
    cin>>n;

    int son= sumOfN(n);
    cout<< son;
}

File: Basics/sumOfN.java
package Basics;
import java.util.*;

public class sumOfN {
    public static int son(int n){
        int sum =0;
        for(int i =0;  i<=n; i++){
            sum+= i;

        }
        return sum;
    }

    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        System.out.println("Eneter the number : ");
        int n = sc.nextInt();
        int sofn = son(n);
        System.out.println("the sum of n numbers is "+ sofn);
    }
}
